buildenv/jenkins/JenkinsfileBase

```diff
@@ -105,8 +105,8 @@ def setupEnv() {
 	env.EXIT_SUCCESS = params.EXIT_SUCCESS ? params.EXIT_SUCCESS : false
 	NUM_MACHINES = params.NUM_MACHINES ? params.NUM_MACHINES.toInteger() : 1
 	env.LIB_DIR = JOB_NAME.contains("SmokeTests") ? "${WORKSPACE}/../../../../../testDependency/lib" : "${WORKSPACE}/../../testDependency/lib"
-	env.OPENJCEPLUS_GIT_REPO = params.OPENJCEPLUS_GIT_REPO ?: "https://github.com/IBM/OpenJCEPlus.git"
-	env.OPENJCEPLUS_GIT_BRANCH = params.OPENJCEPLUS_GIT_BRANCH ?: "java${params.JDK_VERSION}"
+	env.OPENJCEPLUS_GIT_REPO = params.OPENJCEPLUS_GIT_REPO ?: "https://github.com/ibmruntimes/OpenJCEPlus.git"
+	env.OPENJCEPLUS_GIT_BRANCH = params.OPENJCEPLUS_GIT_BRANCH ?: "semeru-java${params.JDK_VERSION}"
 
```

Comment:
This seems correct for a default branch. I expect that by default we should run the tests that are available in the head of the `semeru-javaVERSION` branch for a given java VERSION.

I also assume that this means that the Semeru test code will override this value in a properties file when testing release candidates and final releases instead of using the tests in the head?

Reply:
Yes, by default we will run from `semeru-javaVERSION` branch if the values are not set via job params.

![image](https://github.com/adoptium/aqa-tests/assets/19273206/bcbbd1f6-43b6-47d3-8ee0-161e4f52f6ac)


We will have separate PR for supporting OPENJCEPLUS_GIT_REPO and OPENJCEPLUS_GIT_BRANCH in testenv.propertiesf files.